<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <!-- Example 6 -->
  <node pkg="chapter3_tutorials" type="example6" name="example6"
        output="screen"/>
</launch>

